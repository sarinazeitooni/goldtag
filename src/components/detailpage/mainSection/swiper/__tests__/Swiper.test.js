import React from 'react';
import { shallow } from 'enzyme';
import MainSwiper from "../Swiper";
describe("MainSwiper component" , ()=>{
    it("it should render a div",()=>{
        const container = shallow(<MainSwiper swiperData={['','']}/>)
        expect(container).toMatchSnapshot();
    });
});