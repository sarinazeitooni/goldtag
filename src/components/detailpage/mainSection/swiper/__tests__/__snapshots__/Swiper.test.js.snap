// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainSwiper component it should render a div 1`] = `ShallowWrapper {}`;
